pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -108,11 +118,15 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 
 	var ephemeral bool
 	var device *pmdmanager.PmemDeviceInfo
-	var mountOptions []string
 	var err error
 
 	srcPath := req.GetStagingTargetPath()
 	targetPath := req.GetTargetPath()
+	mountFlags := req.GetVolumeCapability().GetMount().GetMountFlags()
+	readOnly := req.GetReadonly()
+	fsType := req.GetVolumeCapability().GetMount().GetFsType()
+	klog.V(3).Infof("NodePublishVolume request: VolID:%s targetpath:%v sourcepath:%v readonly:%v Capab/mountflags:%v Capab/fsType:%v",
```

Comment:
Can you change `Capab/mountflags` to `mount flags:` in the log message. Capab is not informative.

Reply:
Changed in current code, just pushed